docs/cluster-relocation.md

```diff
@@ -0,0 +1,82 @@
+# Cluster Relocation
+
+Hive supports the ability to migrate a `ClusterDeployment` and it's related custom resources to another Hive cluster through the use of the `ClusterRelocate` custom resource definition.
+
+## Usage
+
+In your source Hive cluster, create a Secret containing a kubeconfig for the destination Hive cluster:
+
+```bash
+$ kubectl create secret generic hub2-migration-kubeconfig --from-file=kubeconfig=./hub2.kubeconfig
```

Comment:
For consistency between this command and the `ClusterRelocate` resource, I recommend specifying the namespace. I also recommend using a namespace other than `hive` so that it does not give the impression that the secret needs to be in the `hive` namespace.
```suggestion
$ kubectl create secret generic hub2-migration-kubeconfig -n hive --from-file=kubeconfig=./hub2.kubeconfig
```

Reply:
Moved to default and added a note on "in the namespace of your choosing"